addons/bus/static/src/js/crosstab_bus.js

```diff
@@ -304,7 +304,7 @@ var CrossTabBus = Longpolling.extend({
 
         // last notification id changed
         if (key === this._generateKey('last')) {
-            this._lastNotificationID = value || -1;
+            this._lastNotificationID = value | 0;
```

Comment:
only one pipe seems wrong.

Reply:
Fixed https://github.com/odoo/odoo/commit/22f037e3af0a2083e4d924657dccd078448311ba
Thanks for feedback :)